srcpkgs/watchman/patches/fix_autogen.patch

```diff
@@ -0,0 +1,13 @@
+From: https://github.com/facebook/watchman/issues/937#issuecomment-904717182
```

Comment:
build still fails with even without this autoconf update:
```
pywatchman/bser.c:31:10: fatal error: Python.h: No such file or directory                                                                         
   31 | #include <Python.h>                                                                                                                       
      |          ^~~~~~~~~~                                                                                                                       
compilation terminated.
```

Reply:
In the autoconf 2.7.1 branch it builds fine: https://paste.rs/Gwu

I didn't had time to try with autoconf 2.6.9

Reply:
With autoconf 2.6.9 also builds: https://paste.rs/wGs

Reply:
Hmm, I must have had a dirty masterdir, I can't reproduce this failure anymore.